src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +22,32 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int NUMBER_OF_CHARS = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        TreeSet<Character> charSet = new TreeSet<>();
+        String finalChars = "";
+
+        try (BufferedReader fileReader = new BufferedReader(new FileReader(fileName))) {
+            while (fileReader.ready()) {
+                String charsOnly = fileReader.readLine().toLowerCase().replaceAll("[^a-zA-Z]", "");
+                for (int i = 0; i < charsOnly.length(); i++) {
+                    charSet.add(charsOnly.charAt(i));
```

Comment:
suggestion: consider using addAll method which takes collection as an argument (think how to convert string charsOnly to collection)